chatcommunicate.py

```diff
@@ -296,7 +296,7 @@ def on_msg(msg, client):
         if result:
             s = ":{}\n{}" if "\n" not in result and len(result) >= 488 else ":{} {}"
             _msg_queue.put((room_data, s.format(message.id, result), None))
-    elif message.content.startswith("!!/"):
+    elif message.content.startswith("!!/") or message.content.lower().startswith("sh "):
```

Comment:
Has this been tested? I don't think it will work because of [the first line of `dispatch_command`](https://github.com/Charcoal-SE/SmokeDetector/blob/81075a1c743be259665c2b28f4af181a647c1758/chatcommunicate.py#L516):

    command_parts = GlobalVars.parser.unescape(msg.content).split(" ", 1)

That splits the command on the first space, separating it into the command name and arguments. If you passed in `sh somecommand args`,  the command will be parsed as `sh` and the arguments will be `somecommand args`, which is not what you want.

Next, the function then strips the first three characters off of the command (to get rid of the `!!/`), which will probably throw an exception or something since the "command" `sh` is only two characters long.